egs/ami/s5b/local/chain/tuning/run_tdnn_1f.sh

```diff
@@ -0,0 +1,266 @@
+#!/bin/bash
+
+# same as 1e but uses batchnorm components instead of renorm also adding
+# proportional-shrink 10, trained with 4 epochs
+
+# System               tdnn1e_sp_bi_ihmali tdnn1f_sp_bi_ihmali
```

Comment:
actually I just realized that you don't specify what microphone these results are with.  I assume sdm, but you should probably state that, and say with what command line options this script was run.